content/en/istio-ca-certs-integrations/cert-manager-integration/_index.md

```diff
@@ -80,53 +84,46 @@ spec:
 EOF
 ```
 
-You can check the `issuers` using the `get issuers` command:
-
+Verify the "istio-ca" and "selfsigned" `issuers` are present and report `READY=True`:
 ```sh
-$ kubectl get issuers -n istio-system
-NAME         READY   AGE
-istio-ca     True    12m
+kubectl get issuers -n istio-system
```

Comment:
include the output to illustrate what should be expected?

Reply:
@incfly I avoid including the output b/c it breaks the copy function. This is why I included the details of the issuers  and expected "READY=True" status above.